src/Core/src/Hosting/EssentialsMauiAppBuilderExtensions.cs

```diff
@@ -27,6 +27,7 @@ internal static MauiAppBuilder UseEssentials(this MauiAppBuilder builder)
 			builder.ConfigureLifecycleEvents(life =>
 			{
 #if __ANDROID__
+				// TODO: Refactor this to not require MauiApplication.Current
 				ApplicationModel.Platform.Init(MauiApplication.Current);
```

Comment:
Why does this not work as `IPlatformApplication`? The api still expects the base android app type?